src/item/TurtleHelmet.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item;
+
+use pocketmine\entity\effect\EffectInstance;
+use pocketmine\entity\effect\VanillaEffects;
+use pocketmine\entity\Human;
+use pocketmine\entity\Living;
+
+class TurtleHelmet extends Armor{
+
+	public function onTickWorn(Living $entity) : bool{
+		if($entity instanceof Human && !$entity->isUnderwater()){
```

Comment:
I'm not sure why this should be limited to humans. It seems reasonable that any entity that can equip it should be able to experience its effects, no?

Reply:
As can seen in the following video, the zombie drowns even wearing the turtle Shell. 

https://user-images.githubusercontent.com/57236932/186192463-e954b6fc-e285-4182-adde-efc6d430fbc8.mp4